bin/check_dependencies

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -uo pipefail
+
+current_dir=$("$(dirname "$0")"/abspath)
+toplevel_dir=$current_dir/..
+
+function finish {
+  rm -f $toplevel_dir/assets/updated_licenses.txt
+}
+trap finish EXIT
+
+# get a new list of project licenses
+# this command syntax is wild but it's what the license_finder image requires
+docker run --rm \
+  -v $toplevel_dir:/scan \
+  licensefinder/license_finder \
+  /bin/bash -lc "cd /scan && license_finder" \
+  > $toplevel_dir/assets/updated_licenses.txt
+
+# check if there are any differences; if not, exit
+diff=$(diff $toplevel_dir/assets/updated_licenses.txt $toplevel_dir/assets/license_finder.txt)
+
+if [ -z "$diff" ]; then
+  echo "No dependencies have changed."
+else
+  echo -e "\nWarning - dependencies have changed. NOTICES.txt will need to be updated.\n"
+  echo "$diff"
+fi
```

Comment:
I don't see any error-type exits here - I'd assume that we eventually want this automated so the exit code of "1" would go a long way to helping automate this even in a simple checking script on Jenkins.

Reply:
This isn't currently part of the pipeline, I don't want it to be until the end to end of generating the file is actually automated, so for now this is a util to check if the txt needs to be updated pre-release.

I'm fine with maybe adding those in later, but right now this is manual and informational.

Reply:
Can you just add a comment or a TODO on this for that then? I'd hate for it to be lost in this PR and never fixed up later.